homeassistant/components/tuya/sensor.py

```diff
@@ -1017,6 +1017,22 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             state_class=SensorStateClass.MEASUREMENT,
         ),
     ),
+    # Soil sensor (Plant monitor)
+    "zwjcy": (
+        TuyaSensorEntityDescription(
+            key=DPCode.TEMP_CURRENT,
+            name="Temperature",
+            device_class=SensorDeviceClass.TEMPERATURE,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.HUMIDITY,
+            name="Humidity",
+            device_class=SensorDeviceClass.HUMIDITY,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        *BATTERY_SENSORS,
+    ),
```

Comment:
Lgtm. Does this addition negatively affect the test coverage?

Reply:
@oliv3r Did you check the coverage before making this comment?

Reply:
I don't think it affects the code coverage; sensor.py of Tuya component get omitted from code coverage.

Reply:
Exactly the point why I asked that @iKaew 👍